otterdog/eclipse-bluechi.jsonnet

```diff
@@ -98,6 +98,32 @@ orgs.newOrg('eclipse-bluechi') {
         },
       ],
     },
+    orgs.newRepo('bluechi-ppa') {
+      allow_update_branch: false,
+      description: "PPA repository for BlueChi's .deb packages",
+      has_projects: false,
+      has_wiki: false,
+      homepage: "https://bluechi.readthedocs.io/en/latest/",
+      secret_scanning: "disabled",
+      secret_scanning_push_protection: "disabled",
```

Comment:
why would you disable secret scanning?

Reply:
Copied this from bluechi repository - but updated the branch just before pushing, so probably an old state. 
Removed both scanning options.